File: cpp/countSeniors.cpp
#include <string>
#include <vector>
using namespace std;

class Solution {
public:
    int countSeniors(vector<string>& details) {
        int count = 0;
        for (string s : details) count += stoi(s.substr(11, 2)) > 60;
        return count;
    }
};

File: rs/countSeniors.rs
impl Solution {
    pub fn count_seniors(details: Vec<String>) -> i32 {
        let mut count = 0;
        for s in details {
            if s[11..13].parse::<i32>().unwrap_or(0) > 60 {
                count += 1;
            }
        }
        count
    }
}